codegen/config/config.go

```diff
@@ -332,6 +333,12 @@ type TypeMapField struct {
 	GeneratedMethod string `yaml:"-"`
 }
 
+type ModelExtraField struct {
+	Type         string `yaml:"type"`
+	IsPointer    bool   `yaml:"isPointer"`
+	OverrideTags string `yaml:"overrideTags"`
```

Comment:
I would ideally like this to contain another string field `Description` that would set the goDoc comment for the field, if possible.